### DISCUSSION TITLE
Importing CSV with Custom 'List' Field

### DISCUSSION THREAD

**Discussion by ijkml** [vendurehq/vendure#576] [Q&A]
I have a customField: **Product:[{ name: 'specs', type: 'string', list: true]**.
When importing a CSV, how should I save the "**product:specs**" column?
Every method I've tried returned an error: **Expected Iterable, but did not find one for field "ProductCustomFields.specs"**.
Please. And thanks.

**Comment by michaelbromley** [comment]
Hi,

This isn't possible at the moment. List types in custom fields are a fairly recent addition, and I didn't yet build in the capability to import them from the CSV format. Created an issue to track this here: https://github.com/vendure-ecommerce/vendure/issues/577
